CSharp/BatchExplorer/Views/MainView.xaml

```diff
@@ -687,12 +658,20 @@
                                         <TextBlock HorizontalAlignment="Center" Text="Delete" FontSize="12" Foreground="Black" />
                                     </StackPanel>
                                 </Button.Content>
-                            </Button>                            
-                        </StackPanel>
+                            </Button>
+                            <TextBox x:Name="SearchJobsTxtBox" 

```

Comment:
Thx. That article seems to be about 2 things.
a) How to control when updates are sent. In our case, we handle update when user Hits Enter which seems pretty standard. 
b) How to bind 1 UI element to another w/o the name. That seems cool but in our code we do required code-behind because we need to send this, correct? Messenger.Default.Send<RefreshMessage>(new RefreshMessage(RefreshTarget.Jobs));